l10n_it_central_journal_reportlab/wizard/print_giornale.py

```diff
@@ -124,6 +126,13 @@ def on_change_daterange_reportlab(self):
             if self.daterange_id.progressive_page_number:
                 self.fiscal_page_base = self.daterange_id.progressive_page_number
 
+    def _clean_aggregated_field_values(self, aggregated_field_values):
```

Comment:
Prob. `@api.model` qui. Non mi sembra che il valore ritornato dipenda da `self`.

In realtà potresti anche metterla in linea, sono
```
", ".join(filter(None, line["partner_names"]))
```
e
```
", ".join(filter(None, line["aml_refs"]))
```
sono abbastanza chiare e compatte (e molto comuni) comunque.